openml-api/README.md

```diff
@@ -0,0 +1,20 @@
+# Feedzai OpenML API
+This module contains the core concepts of the OpenML API.
+
+
+## Developing
+
+To implement an OpenML provider there are two options:
+
+* Implement the **MachineLearningProvider** interface to load trained ML models to the Feedzai platform. It uses the __MachineLearningModelLoader__ interface to load a model.
+
+* Implement the **TrainingMachineLearningProvider** extension of the MachineLearningProvider interface to train new ML models within the Feedzai platform. It uses the __MachineLearningModelTrainer__ interface to train a model.
+
+The other key concepts are the following:
+* __MachineLearningModel__: allows to implement two types of models:
```

Comment:
it doesn't allow to implement two type of models: the MachineLearningModel has two concrete implementations (meaning, it has 2 sub types, or 2 specifications)